src/styles/ComponentPage.scss

```diff
@@ -1,17 +1,18 @@
 @import "@material/layout-grid/dist/mdc.layout-grid";
 @import "@material/list/dist/mdc.list";
+@import "@material/typography/mixins";
 
 .hero {
   display: flex;
   flex-flow: row nowrap;
   align-items: center;
   justify-content: center;
-  height: 360px;
   min-height: 360px;
+  padding: 24px;
   background-color: rgba(0, 0, 0, .05);
 }
 
-.sidebar-active {
+.sidebar-active.mdc-list-item {
```

Comment:
I remember someone saying not to use `.mdc-foo` selectors in the scss. I think just to be safe use a `.demo-` class.

Reply:
Why was a 2nd class added here? I can't tell what this needs to override that it wasn't already overriding.

Reply:
Looks like the font-weight is being overwritten

<img width="280" alt="screen shot 2018-05-01 at 15 58 02" src="https://user-images.githubusercontent.com/579873/39497510-7dd3af92-4d58-11e8-90cc-957e6bcb4ec9.png">